packages/prisma/seed-utils.ts

```diff
@@ -264,3 +265,365 @@ export async function createTeamAndAddUsers(
+                    attributeRaw[2].options.filter((opt) => opt.value === "Sales").map((opt) => opt.id),
+                  ],
+                  operator: "multiselect_some_in",
+                  valueSrc: ["value"],
+                  valueType: ["multiselect"],
+                  valueError: [null],
+                },
+              },
+            },
+          },
+          fallbackAttributesQueryValue: {
+            id: "a9888488-4567-489a-bcde-f19300f53ef8",
+            type: "group",
+          },
+        },
+        {
+          id: "148899aa-4567-489a-bcde-f1823f708646",
+          action: { type: "customPageMessage", value: "Fallback Message" },
+          isFallback: true,
+          queryValue: { id: "814899aa-4567-489a-bcde-f1823f708646", type: "group" },
+        },
+      ],
+      fields: [
+        {
+          id: seededForm.formFieldFilled.id,
+          type: "multiselect",
+          label: "skills",
+          options: attributeRaw[2].options.map((opt) => ({
+            id: opt.id,
+            label: opt.value,
+          })),
+          required: true,
+        },
+      ],
+      team: {
+        connect: {
+          id: teamId,
+        },
+      },
+      user: {
+        connect: {
+          id: userId,
+        },
+      },
+      name: seededForm.name,
+    },
+  });
+  return seededForm;
+}
+
+type SeededForm = {
+  id: string;
+  name: string;
+  routes: {
+    id: string;
+    value: string;
+  }[];
+  formFieldFilled: {
+    id: string;
+  };
+};
+
+// id	formFillerId	formId	response	createdAt	routedToBookingUid
+// 1	cm35rrf1x0001rqjskhydev9i	948ae412-d995-4865-885a-48302588de03	"{""83316968-45bf-4c9d-b5d4-5368a8d2d2a8"": {""label"": ""skills"", ""value"": [""8d841a71-3b96-4930-9e2d-57a3584ec21b""]}}"	2024-11-06 11:01:56.853
+
+export async function seedRoutingFormResponses(
+  seededForm: SeededForm,
+  attributeRaw: { id: string; options: { id: string; value: string }[] }[],
+  teamId: number
+) {
+  // Get all bookings for this team
+  const bookings = await prisma.booking.findMany({
+    where: {
+      eventType: {
+        teamId: teamId,
+      },
+    },
+    take: 250, // Limit to 1000 responses
+  });
+
+  if (bookings.length === 0) {
+    console.log("No bookings found for team - skipping routing form responses");
+    return;
+  }
+
+  // Create routing form responses linked to random bookings
+  const responses = bookings.map((booking) => {
+    // Randomly select 1-3 skills from the form field options
+    const numSkills = Math.floor(Math.random() * 3) + 1;
+    const shuffledOptions = [...attributeRaw[2].options].sort(() => Math.random() - 0.5);
+    const selectedSkills = shuffledOptions.slice(0, numSkills);
+
+    const formFillerId = randomUUID();
+
+    return {
+      formId: seededForm.id,
+      formFillerId: formFillerId,
+      response: {
+        [seededForm.formFieldFilled.id]: {
+          label: "skills",
+          value: selectedSkills.map((opt) => opt.id),
```

Comment:
We can pass multiple responses in here Note: These responses may not match the rules that are created. They are just for visual and display purposes for insights.